packages/gestalt/src/ToastPrimaryAction.js

```diff
@@ -0,0 +1,43 @@
+// @flow strict
+import { type Node } from 'react';
+import Button from './Button.js';
+import Link from './Link.js';
+
+type Props = {|
+  accessibilityLabel?: string,
```

Comment:
`accessibilityLabel` and `label` should be required, no?

Reply:
DONE